relayer/src/common/events/polygon-zkevm/BridgeEvent.ts

```diff
@@ -0,0 +1,54 @@
+import type { Log } from "@ethersproject/providers";
+import type { BigNumber } from "ethers";
+
+import { PolygonZKEvmBridge__factory } from "../../../../types-gen/contracts";
+import { isUndefined } from "../../util";
+
+interface Result extends ReadonlyArray<unknown> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any;
```

Comment:
Why are we using `any` here? What could be the possible values be? boolean, bytes, string?

Reply:
Literally anything. Whatever value ethers parses for the event parameter values.